pkg/cmd/pulumi/convert.go

```diff
@@ -94,6 +134,10 @@ func runConvert(cwd string, language string, outDir string, generateOnly bool) r
 		projectGenerator = javagen.GenerateProject
 	case "yaml": // nolint: goconst
 		projectGenerator = yamlgen.GenerateProject
+	case "pulumi", "pcl":
```

Comment:
I would consider adding `fmt.Fprintf(os.Stderr, "Warning: PCL is an implementation feature and not stable\n")` or gating behind a env var to make sure people know this is not for normal use.

Reply:
PULUMI_DEV is used elsewhere to silence warnings like this

Reply:
I've updated the code such that this only runs when `PULUMI_DEV` is present and is truthy ✅